新增多语言的 PR (#66)
WForst-Breeze/PCL2-1930: README.md

 | **支持通过启动器联机游玩 Minecraft** | ***“至于联机，因为网易还在继续代理中国版，我和 HMCL 的维护者都觉得还得谨慎一点……况且现在也有 Mod 支持联机了……咳咳。”*** <br> ***“目前联机并不是遇到了技术问题。等到网易不代理国服了联机就有了”*** <br> <div align="right"> ***——龙腾猫跃*** </div> <br> *BakaXL 现已于 2023 年 10 月 8 日将其部分服务与 Octo 章鱼网络进行对接，并对其可用性进行初步评估，待评估完成后，Octo 网络会全面对其他启动器开放。*[^4]  | **[2.6.11&nbsp;更新日志](https://www.bilibili.com/read/cv28121157/)** #4166 [#4158](https://github.com/Hex-Dragon/PCL2/discussions/4158#discussioncomment-9968069)
 | PCL 消失、报毒、无法更新 | 为 Microsoft Defender 和 360 等杀毒软件误报，请在 Windows 安全中心和杀毒软件中添加排除项或白名单。具体教程见 PCL 内置帮助库。 |  PCL2&nbsp;内置帮助库 <br> #56 [#666](https://github.com/Hex-Dragon/PCL2/issues/666#issuecomment-1611534587)
 | 以管理员权限运行时无法使用文件拖拽功能 | **[需要社区帮助]** *“尽可能尝试修复相关代码，依然无法解决……如果谁有解决方案可以提出”* ——龙腾猫跃 | [#2531](https://github.com/Hex-Dragon/PCL2/issues/2531#issuecomment-1822511620)

>  | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Voting in Discussions, a related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)

Reviewer: ```suggestion
 | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· It is voting in Discussions. A related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)
```

>  | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Voting in Discussions, a related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)

Reviewer: ```suggestion
 | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· It is being Voted in Discussions. A related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)
```

>  | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Voting in Discussions,there is already a related Pull Request. | 🔹 **#4145** <br> #891 (#2567)

Reviewer: ```suggestion
 | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Voting in Discussions, there is already a related Pull Request. | 🔹 **#4145** <br> #891 (#2567)
```

>  | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Being voted in Discussions. A related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)

Reviewer 2: 英文为：Voting in Discussions. Related Pull Request already exists. 
会不会更好一点？以精炼易懂为主即可

>  | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· It is being Voted in Discussions. A related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)

Reviewer: ```suggestion
 | 支持启动器多语言选项<br>· I18n support & Support more languages | 正在 Discussions 中投票，已有相关 Pull Request。<br>· Being voted in Discussions. A related Pull Request already exists. | 🔹 **#4145** <br> #891 (#2567)
```
